README.md

```diff
@@ -127,6 +127,15 @@ for use with background images
 </div>
 ```
 
+If you want to load specific images before they appear:
+
+```js
+const observer = lozad();
+observer.observe();
+
+// ... trigger the load of a specific image
+observer.triggerLoad('.specific-selector'); // this will load the image that satisfies the .lozad.specific-selector selector
```

Comment:
this should load the image that satisfies `.specific-selector` instead of `.lozad.specific-selector`